### DISCUSSION TITLE
use headless mode to goto https url, return ERR_BAD_SSL_CLIENT_AUTH_CERT

### DISCUSSION THREAD

**Discussion by hw64220** [getgauge/taiko#2686] [General]
**Describe the bug**
I AM not sure whether its a bug. I knew it works ok if I use a public website such as google for testing .
So, the key problem should be  we are using the self-signed certification, any suggestion or guide on such situation? 

**To Reproduce**
Steps (or script) to reproduce the behavior:
1.we make the certification by ourselves(server and client).
2.the certification is self-signed and  trusted by browser 
3.user below scripts to run with headless mode, will return such error
> openBrowser({headless: true,ignoreCertificatieErrors:true})
[PASS] Browser opened
> goto("https://service.cffex.com.cn:8443/ecms-frontend-ext/")
[FAIL] Error: Navigation to url https://service.cffex.com.cn:8443/ecms-frontend-
ext/ failed. REASON: net::ERR_BAD_SSL_CLIENT_AUTH_CERT, run `.trace` for more in
fo.
```

**Logs**
> .trace
'Error: Navigation to url https://service.cffex.com.cn:8443/ecms-frontend-ext/ f
ailed. REASON: net::ERR_BAD_SSL_CLIENT_AUTH_CERT
' +
  '    at Object.handleNavigation (C:\\Users\\Administrator\\AppData\\Roaming\
pm\
ode_modules\\taiko\\lib\\handlers\\pageHandler.js:193:13)
' +
  '    at async C:\\Users\\Administrator\\AppData\\Roaming\
pm\
ode_modules\\taiko\\lib\\taiko.js:745:16
' +
  '    at async doActionAwaitingNavigation (C:\\Users\\Administrator\\AppData\\R
oaming\
pm\
ode_modules\\taiko\\lib\\doActionAwaitingNavigation.js:61:5)
' +
  '    at async module.exports.goto (C:\\Users\\Administrator\\AppData\\Roaming\

pm\
ode_modules\\taiko\\lib\\taiko.js:744:3)
' +
  '    at async module.exports.<computed> (C:\\Users\\Administrator\\AppData\\Ro
aming\
pm\
ode_modules\\taiko\\lib\\taiko.js:2620:14)
' +
  '    at async repl.context.<computed> (C:\\Users\\Administrator\\AppData\\Roam
ing\
pm\
ode_modules\\taiko\\lib\\repl\\repl.js:271:23)
' +
  '    at async C:\\Users\\Administrator\\AppData\\Roaming\
pm\
ode_modules\\taiko\\lib\\repl\\awaitEval.js:54:17'

**Comment by zabil** [comment]
There is typo in the posted code sample. Try changing

```
openBrowser({headless: true,ignoreCertificatieErrors:true})
```

to 

```
openBrowser({headless: true,ignoreCertificateErrors:true})
```

**Reply by hw64220** [reply]
thx @zabil . My bad, there is typo.
But on the other hand,if my apache open Two-Way SSL authentication(SSLVerifyClient require), it still failed for this error.
If I turn off the Two-Way SSL authentication, it works.
